<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportsWidget</class>
 <widget class="QWidget" name="ReportsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reports</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="ReportParamsFrame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="10">
       <spacer name="ReportFrameSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="AccountButton" name="ReportAccountBtn"/>
      </item>
      <item row="0" column="9">
       <widget class="CategorySelector" name="ReportCategoryEdit" native="true"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ReportTypeLbl">
        <property name="text">
         <string>Report:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDateEdit" name="ReportToDate">
        <property name="dateTime">
         <datetime>
          <hour>21</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2020</year>
          <month>11</month>
          <day>25</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="timeSpec">
         <enum>Qt::UTC</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ReportAccountLbl">
        <property name="text">
         <string>Account:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="Line" name="line_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="ReportToLbl">
        <property name="text">
         <string>To:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="ReportFromLbl">
        <property name="text">
         <string>From:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QCheckBox" name="ReportGroupCheck">
        <property name="text">
         <string>Group dates</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="ReportTypeCombo">
        <item>
         <property name="text">
          <string>Income / Spending</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Profit / Loss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Deals</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>By Category</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="11">
       <widget class="QPushButton" name="SaveReportBtn">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QPushButton" name="RunReportBtn">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="ReportCategoryLbl">
        <property name="text">
         <string>Category:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QComboBox" name="ReportRangeCombo">
        <item>
         <property name="text">
          <string>Custom range</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quarter to date</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year to date</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>This year</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Previous year</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDateEdit" name="ReportFromDate">
        <property name="dateTime">
         <datetime>
          <hour>21</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2020</year>
          <month>11</month>
          <day>25</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="timeSpec">
         <enum>Qt::UTC</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="ReportTableView">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::DotLine</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="ReportTreeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccountButton</class>
   <extends>QPushButton</extends>
   <header>jal/widgets/account_select.h</header>
  </customwidget>
  <customwidget>
   <class>CategorySelector</class>
   <extends>QWidget</extends>
   <header>jal/widgets/reference_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
